recipes/triqs_cthyb/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "2.2.1" %}
+
+package:
+  name: triqs_cthyb
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/cthyb/releases/download/{{ version }}/cthyb-{{ version }}.tar.gz
+  sha256: de78b39a354c0e1344987c26403919c23611635ab3d75620aa01bd5b06431bdc
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
```

Comment:
No python3 support? We are about to drop python2 support.

Reply:
The python3 ports are in the works and should be finished within the next two to three weeks.
I will then update the feedstocks accordingly.